

##### plants Article Genome-Wide Association Study Reveals the Genetic Basis of Kernel and Cob Moisture Changes in Maize at Physiological Maturity Stage 
Minyan Zhang † , Chaoyang Xiangchen † , Jiaquan Yan, Yujuan Chengxu, Hao Liu, Chaoying Zou, Guangtang Pan, Yaou Shen and Langlang Ma * 
State Key Laboratory of Crop Gene Exploration and Utilization in Southwest China, Maize Research Institute, 
Sichuan Agricultural University, Chengdu 611130, China; minyan_z@163.com (M.Z.); 
xccy498720303@163.com (C.X.); yjq15082436449@163.com (J.Y.); ccxxyyjj57364@163.com (Y.C.); 
liuhao971229@163.com (H.L.); zoucy2022@163.com (C.Z.); pangt@sicau.edu.cn (G.P.); shenyaou@sicau.edu.cn (Y.S.) * Correspondence: sxyljxml@163.com † These authors contributed equally to this work. 
Abstract: Low moisture content (MC) and high dehydration rate (DR) at physiological maturity 
affect grain mechanical harvest, transport, and storage. In this study, we used an association panel 
composed of 241 maize inbred lines to analyze ear moisture changes at physiological maturity stage. 
A genome-wide association study revealed nine significant SNPs and 91 candidate genes. One SNP 
(SYN38588) was repeatedly detected for two traits, and 15 candidate genes were scanned in the 
linkage disequilibrium regions of this SNP. Of these, genes Zm00001d020615 and Zm00001d020623 
were individually annotated as a polygalacturonase (PG) and a copper transporter 5.1 (COPT5.1), 
Citation: Zhang, M.; Xiangchen, C.; respectively. Candidate gene association analysis showed that three SNPs located in the exons 
Yan, J.; Chengxu, Y.; Liu, H.; Zou, C.; of Zm00001d020615 were significantly associated with the dehydration rate, and AATTAA was Pan, G.; Shen, Y.; Ma, L. 
determined as the superior haplotype. All these findings suggested that Zm00001d020615 was a key Genome-Wide Association Study 
gene affecting moisture changes of maize at the physiological maturity stage. These results have Reveals the Genetic Basis of Kernel 
demonstrated the genetic basis of ear moisture changes in maize and indicated a superior haplotype and Cob Moisture Changes in Maize at Physiological Maturity Stage. 
for cultivating maize varieties with low moisture content and high dehydration rates. Plants 2022, 11, 1989. https:// 
doi.org/10.3390/plants11151989Keywords: maize; physiological maturity; moisture content; dehydration rate; genome-wide association study Academic Editors: Junjie Zou, Miaoyun Xu and Yong-Gu Cho Received: 8 July 2022 Accepted: 28 July 20221. Introduction 
Published: 30 July 2022Maize (Zea may L.) is one of the main food crops worldwide and is an important source 
Publisher’s Note: MDPI stays neutralof human nutrition, animal feed, and bioenergy. To further improve production efficiency 
with regard to jurisdictional claims in and reduce production costs, mechanized grain harvesting has become a key technology for 
published maps and institutional affil- crop production [1]. However, ears having a high moisture content (MC) at the harvesting 
iations.stage causes grain mildew and restrict mechanized harvesting [2]. In the natural drying 
process from physiological maturity to harvest, the initial MC at physiological maturity 
and the dehydration rate (DR) in the field environment jointly determine the final MC at 
harvest [3,4]. Therefore, breeding maize hybrids with low MC and high DR at physiological 
Copyright: © 2022 by the authors. maturity can promote mechanical harvest, improve threshing efficiencies, and reduce 
Licensee MDPI, Basel, Switzerland.additional drying costs [5–7]. This article is an open access article 
 The MC and DR are quantitative traits mainly controlled by genetic factors [8], al- distributed under the terms and 
though they are affected by changes in the field ambient temperature and humidity [9,10]. conditions of the Creative Commons 
Currently, numerous moisture-gain-associated QTL have been detected by linkage analysis. Attribution (CC BY) license (https:// 
For example, a total of 76 QTL associated with dehydration characteristic parameters were creativecommons.org/licenses/by/ 
detected using a recombinant inbred line (RIL) population of 208 lines, which explained 
4.0/). 
Plants 2022, 11, 1989. https://doi.org/10.3390/plants11151989https://www.mdpi.com/journal/plants  

##### Plants 2022, 11, 19892 of 11 
1.03% to 15.24% of the phenotypic variation [11]. Liu et al. [12] identified seven QTL related 
to grain water content using an RIL population, which explained 6.92–24.78% of the pheno- 
typic variation. However, only a few moisture-gain-associated genes were fine-mapped due 
to the relatively low resolution of linkage mapping [13]. Genome-wide association study 
(GWAS) is an effective tool for analyzing the genetic structure of complex quantitative traits, 
which provides a high-resolution strategy for identifying the loci of quantitative traits [14]. 
The genetic controls of many agronomic traits have been identified by GWAS [15–24]. For 
example, in recent years, several SNPs related to MC and DR were detected by GWAS 
at different stages of maize kernel development. Using an association panel containing 
513 maize inbred lines, Li et al. [25] identified 71 SNPs affecting MC by GWAS. Combined 
with genetic population analysis, transcriptome analysis, and gene editing, two key genes 
Zm00001d020929 and Zm00001d046583 were shown to negatively regulate the MC. A total 
of 16 SNPs significantly associated with MC were detected by GWAS among a natural panel 
consisting of 310 maize inbred lines at the harvest stage [26]. Using GWAS, Li et al. [27] 
identified 27 SNPs related to kernel water content (KWC) and kernel dehydration rate 
(KDR) in maize. In addition, 334 SNPs were detected that significantly controlled the 
kernel moisture content (KMC) and KDR in 132 maize inbred lines before physiological 
maturity [28]. Despite the conduct of these studies, the genetic control of maize kernel and 
cob moisture changes at the physiological maturity stage remains unclear. 
In this study, we measured cob moisture content (CMC), cob dehydration rate (CDR), 
KMC, and KDR values of 241 maize inbred lines on the 45th, 50th, and 55th days after pol- 
lination (DAP). GWAS was performed using the fixed and random circulating probability 
unification (FarmCPU) model to uncover the important SNPs and causal genes that control 
MC and DR. In addition, candidate gene association studies were conducted to elucidate 
the intragenic variations affecting the target traits. This study sought to (i) reveal the genetic 
basis of moisture change in maize kernel and cob, (ii) identify the hub genes affecting the 
maize moisture change, and (iii) contribute to the development of function markers for 
breeding low-MC and high-DR maize varieties through marker-assisted selection. 
2. Results 
2.1. Phenotypic Descriptions 
 The 12 traits investigated in the present study displayed great variation in the asso- 
ciation panel of 241 maize inbred lines (Table 1). Most of the 12 traits followed normal 
distributions (Figure S1). At 45, 50, and 55 DAP, the mean values of CMC and KMC 
ranged from 57.01% to 62.60% and 26.98% to 35.70%, respectively (Table 1). The CMC and 
KMC both significantly declined from 45 to 55 DAP (Figure 1A). For DR at two successive 
time spans, the means of CDR and KDR were 0.95%–1.11% and 0.85%–1.16%, respectively 
(Table 1). The CDR and KDR variations both significantly (p < 0.05) increased from 45 to 
55 DAP (Figure 1B). The coefficients of variation (CVs) of these12 traits were in the range 
of 0.61% to 10.68% (Table 1). In addition, the mean value of broad-sense heritability (H2 ) 
was 57.34% (Table 1), which indicated that the MC-related traits were mainly controlled by genetic factors. 
Table 1. Phenotypic variations of 12 traits in 241 maize inbred lines. Trait Mean (%)Max (%)Min (%)SD CV (%)H 2 (%)  CMC45 62.6078.75 46.37 0.380.6162.38  CMC50 61.2677.28 42.30 0.480.7973.69  CMC55 57.0171.80 35.52 0.711.2565.25  KMC45 35.7073.50 22.52 0.491.3729.79  KMC50 32.5965.55 14.46 0.491.5035.15  KMC55 26.9846.72 11.46 0.612.2668.71  

##### Plants 2022, 11, 19893 of 12 Plants 2022, 11, 1989 3 of 11  KMC5032.59 65.55 14.460.49 1.5035.15  KMC5526.98 46.72 11.460.61 2.2668.71  CDR45–50 0.95 4.780.010.08 8.1592.16 Table 1. Cont.  CDR50–55 1.11 4.530.020.10 9.3588.41 TraitMean (%) Max (%)Min 0.01  (%) SDCV(%) 2 (%)  H60.41  CDR45–55 0.72 3.410.0810.68  KDR45–50 CDR45–50 0.950.854.781.840.06 
0.01 0.04 
 0.08 7.51 
 8.15 22.20 92.16 CDR50–55  KDR50–551.111.164.532.56 0.02 
 0.050.10 
 0.089.35 
8.7588.41 24.52 CDR45–55 0.723.41 0.01 0.08 10.68 60.41  KDR45–55 0.75 1.360.060.04 6.3365.41 KDR45–50 0.85 1.840.06 0.04 7.51 22.20 Max, maximum; Min, KDR50–55 1.16minimum; 2.56SD, standard deviation; 
0.05coefficient of variation;  CV, 0.08 8.75 broad-  H2, 24.52 senseheritability. CMC45, KDR45–55 0.75CMC50,and CMC550.06 
1.36represent cob0.04 moisture content6.33 on the 45th, 50th,  65.41 
and 55th days after pollination, respectively; KMC45, KMC50, and KMC55 represent 2 kernel mois- 
Max, maximum; Min, minimum; SD, standard deviation; CV, coefficient of variation; H , broad-sense heritability. ture content CMC45, CMC50,the 45th,  on andCMC55 50th, and 55th  represent days after  cob moisturepollination,  contentrespectively; on the 45th,CDR45–50,  50th, and 55th days afterCDR50–55, pollination, and CDR45–55 respectively;represent KMC45,KMC50, cobdehydration and 45–50,  rate atkernel KMC55 representmoistureand 45–55  50–55, content days  on the after  45th,pollination,  50th, and 55th days after pollination, respectively;respectively; KDR45–50, CDR45–50,and  KDR50–55,CDR50–55,  KDR45–55represent  CDR45–55kernel  andrepresent cob dehydration dehydration rate rate atat 45–50, 45–50, 50–55, 50–55,and and45–55  45–55days  daysafter pollination,  afterrespectively; pollination,KDR45–50, KDR50–55, and KDR45–55 represent kernel respectively. 
dehydration rate at 45–50, 50–55, and 45–55 days after pollination, respectively. Phenotypesofofmoisture-content-related Figure 1. Phenotypesmoisture-content-related traits of of traits thethe association  associationpanel panel at physiological at physiological ma- turity stages. maturity(A) (A)  stages.Phenotypic  Phenotypicvaluesof CMC values of CMC KMC and and KMCmeasured at each measured stage.  at each stage.moisture MC, MC,con-  moisture content. Phenotypic tent. (B) (B) Phenotypic values valuesof of  CDRCDRand  andmeasuredfor KDRmeasured KDR thetwo forthetwotime DR, dehydration  spans. DR,  time spans.dehydration rate. * Significantat p < 0.05. ** Significant at p < 0.01. rate. * Significant at p < 0.05. ** Significant at p < 0.01. 
2.2. SNPs and Candidate Genes Associated with Moisture ChangesChanges  GWASwas GWASwasperformed  performedforfor  MCMCandand DR toDRexplore the genetic  to exploremoisture  basis ofbasis the genetic changes.  of moisture Atotal ofA changes. four,  totalthree,andthree,  three significant  of four,(p < 1.82 ×(p10<−1.82  and three significant 6 ) SNP markers × 10−6) SNP weremarkersdetected were for CDR45–55, detected CDR50–55, for CDR45–55, and KMC45,  CDR50–55,and respectively, which distributed KMC45, respectively, on chromosomes  which distributed on chro- 1, 2, 3, 5, and mosomes1, 2,7 3, (Figure 5, and2 7andTable2S1). andAmong (Figurethese Table S1).significant  theseSNPs,  Amongone (SYN38588)  significant SNPs, one was simultaneouslyassociated with CDR50–55 and 
(SYN38588) was simultaneously associated with CDR50–55 and CDR45–55CDR45–55andhad thelowest and hadp-value the 
(p =9.58× 10 −11 ) in CDR45–55associations (Table S1).Basedonourprevious reports, the 
lowest p-value (p = 9.58 × 10 ) in CDR45–55 associations (Table S1). Based on our previous −11 linkage reports, disequilibrium(LD) decay was  the linkage disequilibrium(LD)around decay220waskbaround  for this220maizepanel  kb forthis[29].Thus, maize we  panel searched [29]. Thus,thewegene models searchedthe in geneLD regions themodels in theofLDtheregionsof theSNPs. significantIn total,  significantSNPs.weInfound total, wemoisture-change-associated 91 found 91 moisture-change-associatedgenes (Tablesgenes 2 and(Tables S2). For2CDR50–55 and S2). ForCDR45–55, and and CDR50–55one SNP, SYN38588, was repeatedly detected(TableS1).Withinthe LD 
CDR45–55, one SNP, SYN38588, was repeatedly detected (Table S1). Within the LD region region of the co-detected marker(SYN38588),marker of the co-detected 15 gene(SYN38588),  models were15uncovered, gene models of whichfour were functionally  were uncovered,of which fouran- notatedandtheother 11 were unknown genes in maize (Table 
were functionally annotated and the other 11 were unknown genes in maize (Table 2). 2). Notably,thetwo genes Zm00001d020615 Notably, the two and genesZm00001d020623  Zm00001d020615 separatelyencode polygalacturonase  and Zm00001d020623 separately encode(PG) andcop- polyga- pertransporter lacturonase(PG) (COPT5.1), 
 5.1and respectively.5.1 copper transporterAccording (COPT5.1), to previous  respectively.studies,PG and COPT5.1 Accordingto previ- proteinsare involvedinregulatingtheMCand DRof 
ous studies, PG and COPT5.1 proteins are involved in regulating the MC and soybeanand rice seeds  DR[30–34]. of soy- 
To further understand the functions of the 11 genes, we performed a homology analy- 
bean and rice seeds [30–34]. To further understand the functions of the 11 genes, we per- 
sis and found six homologous genes in rice, sorghum, and Panicum virgatum (Table S3). 
formed a homology analysis and found six homologous genes in rice, sorghum, and Pani- 
However, the encoding proteins of these homologous genes had not been reported to be 
cum virgatum (Table S3). However, the encoding proteins of these homologous genes had 
associated with moisture changes. Therefore, Zm00001d020615 and Zm00001d020623 were 
not been reported to be associated with moisture changes. Therefore, Zm00001d020615 
selected as the prioritized candidate genes that controlled the moisture changes of maize at physiological maturity.  

##### Plants 2022, 11, 1989 4 of 12 Plants 2022, 11, 19894 of 11 
and Zm00001d020623 were selected as the prioritized candidate genes that controlled the moisture changes of maize at physiological maturity. Figure 2. 
2. Significant SNPs detected by GWASGWAS using  using FarmCPU  FarmCPU model. model. (A) Manhattan  Manhattan diagram of 
GWASresultsfor moisturechanges.The blue, red, and turquoise dotsrepresent 
GWAS results for moisture changes. The blue, red, and turquoise dots represent the the significant SNPs  significant SNPs 
associatedwith CDR50–55,CDR45–55, and KMC45,respectively.CDR50–55 and CDR45–55 
associated with CDR50–55, CDR45–55, and KMC45, respectively. CDR50–55 and CDR45–55 representrepre- 
sent cob dehydration rate at 50–55 and 45–55 days after pollination, respectively; KMC45 represents 
cob dehydration rate at 50–55 and 45–55 days after pollination, respectively; KMC45 represents kernel 
kernel moisture content on the 45th day after pollination. The red broken line represents the signif- 
moisture content on the 45th−6day after pollination. The red broken line represents the significant 
icant threshold of 1.82× 10 . (B) Quantile–quantile (Q–Q) plot of GWAS results for moisture 
−6 . (B) Quantile–quantile (Q–Q) plot of GWAS results for moisture changes. The thresholdof 1.82 changes. The × 10 
blue, green,and yellow dots represent the SNPs associated with CDR50–55, CDR45– blue, green, 55, and KMC45,and yellow dots represent  respectively.CDR50–55SNPs theand associated  CDR45–55 with CDR50–55,  representrate and CDR45–55, cob dehydration KMC45,  at 50–55 and respectively.CDR50–55and CDR45–55representcob dehydration 
45–55 days after pollination, respectively. KMC45 represents kernel moisture content on the after rate at 50–55 and45–55days45th 
pollination, respectively. KMC45 represents kernel moisture content on the 45th day after pollination. day after pollination. Table 2. Table Annotationsofofcandidate candidategenes 
 2. Annotations genesscanned from  scanned co-localized fromsignificant  co-localizedsignificant(SYN38588) SNPSNP de-  (SYN38588) tected byGWAS. detected by GWAS. AssociatedTraits Associated TraitsCandidate Genes Candidate GenesAnnotations  Annotations  Zm00001d020618 Zm00001d020618- -  Zm00001d020610 Zm00001d020610 -- Zm00001d020609 -  Zm00001d020609 - Zm00001d020612  Zm00001d020612 - - Zm00001d020622-  Zm00001d020622 Zm00001d020615 - polygalacturonase  CDR45–55Zm00001d020615 Zm00001d020626 polygalacturonase -  CDR45–55  CDR50–55Zm00001d020626 Zm00001d020613 --  CDR50–55 Zm00001d020617 -  Zm00001d020613 exocyst complex component EXO70B1 Zm00001d020614-  Zm00001d020617 exocyst complex component EXO70B1 Zm00001d020623copper transporter 5.1  Zm00001d020614 Zm00001d020616 --  Zm00001d020623 Zm00001d020627 copper-transporter 5.1 Zm00001d020628 -  Zm00001d020616 trimethyltridecatetraene synthase Zm00001d020620  Zm00001d020627 - - 
 CDR45–55 and CDR50–55 represent cob dehydration rate at 45–55 and 50–55 days after pollination, respectively.  Zm00001d020628 ‘-’ represents no functional annotations. trimethyltridecatetraene synthase  Zm00001d020620 - CDR45–55 
2.3. andGene  Candidate CDR50–55represent  Association cob dehydration  Analysis Revealed Intragenic 50–55 days  45–55 and Affecting  rate at Variationsafter pollina-  MoistureChanges tion, respectively. ‘-’ represents no functional annotations. 
To identify the causal gene(s) that control MC and DR at the physiological maturity 
stage in maize, we performed candidate gene association analyses for Zm00001d020615 and 
Zm00001d020623 using 67 randomly selected lines from the maize association panel. We 
detected 29 (27 SNPs and two indels) and nine (eight SNPs and one indel) polymorphic 
sites in the promoter (upstream 2000 bp), UTR, and gene body regions of Zm00001d020615 
and Zm00001d020623, respectively (Table S4). Candidate gene association analyses showed  


##### Among the three significant SNPs detected in Zm00001d020615, one missense varia- 
 tion (C/A) occurred in the fifth exon, and the other two synonymous variations (G/A, C/T) 
 were located in the third and fifth exons, respectively (Figure 3A, Table 3). The 67 inbred 
 lines were classified into three haplotypes (Hap1: AATTAA, Hap2: GGCCAA and Hap3: 
Plants 2022, 11, 1989GGCCCC) based on the three variations. The lines with Hap1 had a significantlyhigher  5 of 11 
 CDR50–55 value than those with Hap 3 (p < 0.05), whereas no significant difference in 
 moisture-change-related phenotypes was observed between Hap2 and each of Hap1 and  Hap3 that(Figure  three significant such, (p 3B). As SNPsHap1= 2.63 × 10−3as  was considered < 0.05/19 the  ) in superior haplotype Zm00001d020615 in this study. were identified,  Allinvolving  these findingssuggest two SNPsthat Zm00001d020615 (S7_125166053 is a key and S7_125166495)gene affecting  relatedmoisture  to KDR45–50,and onechanges CDR50–55-associated  of maizeSNP (S7_125165191) 
at the physiologicalmaturity stage.(Figure 3A, Table 3). However, no significant variation loci were identified for Zm00001d020623.  Figure3. 3. Figure Associationanalysis  analysis of Zm00001d020615. AssociationZm00001d020615.(A) (A) Significant Significant SNPsSNPs  associatedwith with associatedCDR50–55  CDR50–55 KDR45–50. KDR45–50.CDR50–55  andand representcob  CDR50–55 represent cobdehydration  dehydrationraterate at 50–55 days days  at 50–55after pollination; after pollination; KDR45–50KDR45–  50 represent kerneldehydration represent kerneldehydrationrate rate at at 45–50 45–50 days  days after after pollination.Red Red  pollination.line shows  line shows the significant the significant  threshold thresholdofof markers.The markers. The boxes  boxes withwithgray, red, white,  gray, white, and red, colorsblack  black and colors representrepresent  promoterpromoter  (upstream 
2000 bp), UTR, exons, and introns, respectively. Bottom plot shows the pairwise linkage disequilib- 
rium between the target SNP markers. (B) Comparison of three haplotypes for CDR50–55. CDR50–55 
represent cob dehydration rate at 50–55 days after pollination. Hap1, haplotype1; Hap2, haplotype2; 
Hap3, haplotype3. * Significant at p < 0.05. ns, no significant. 
Table 3. Gene-based association studies detected significantly associated variants within Zm00001d020615. Trait Marker Positionp-ValueAllele Variation Region 
CDR50–55S7_1251651911251651910.00259 G/A the third exon (synonymous) 
KDR45–50S7_1251660531251660530.00246 C/Tthe fifth exon (synonymous) S7_1251664951251664950.00108 C/Athe fifth exon (missense) 
CDR50–55 represent cob dehydration rate at 50–55 days after pollination; KDR45–50 represent kernel dehydration rate at 45–50 days after pollination. 
Among the three significant SNPs detected in Zm00001d020615, one missense variation 
(C/A) occurred in the fifth exon, and the other two synonymous variations (G/A, C/T) 
were located in the third and fifth exons, respectively (Figure 3A, Table 3). The 67 inbred  


#####  Table 3. Gene-based association studies detected significantly associated variants within  Zm00001d020615.  Trait MarkerPositionp-Value AlleleVariation Region Plants 2022, 11, 1989 CDR50–55S7_1251651911251651910.00259G/A 6 of 11 the third exon (synonymous) 
KDR45–50S7_1251660531251660530.00246C/Tthe fifth exon (synonymous) S7_1251664951251664950.00108C/Athe fifth exon (missense) 
lines were classified into three haplotypes (Hap1: AATTAA, Hap2: GGCCAA and Hap3: 
 CDR50–55 represent cob dehydration rate at 50–55 days after pollination; KDR45–50 represent 
GGCCCC) based on the three variations. The lines with Hap1 had a significantly higher  kernel dehydration rate at 45–50 days after pollination. 
CDR50–55 value than those with Hap 3 (p < 0.05), whereas no significant difference in 
moisture-change-related phenotypes was observed between Hap2 and each of Hap1 and 
 2.4.Hap3 Distributionsof Superior Alleles in Elite Lines 
 (Figure 3B). As such, Hap1 was considered as the superior haplotype in this study. AllAmongthe association these findings panel,  suggest that 30 elite lines Zm00001d020615is awith excellent  key gene moisturetraits agronomic affectinghave  of been  changes maizeused  widely at theasphysiological maturity  the parent lines stage.  for cultivating commercial maize varieties [29]. Evaluation 
 of the utilization of superior alleles for the significant SNPs controlling the moisture- 
2.4. Distributions of Superior Alleles in Elite Lines 
 change-related traits is urgently needed for the breeding of varieties with low MC and 
 Among the association panel, 30 elite lines with excellent agronomic traits have been 
 high DR. Herein, the alleles that were associated with higher DR and lower MC values 
widely used as the parent lines for cultivating commercial maize varieties [29]. Evaluation 
 weredefined as the superior alleles; conversely, the alleles associated with lower DR and 
of the utilization of superior alleles for the significant SNPs controlling the moisture-change-  higher relatedMC values traits were designated needed for as  is urgentlythetheinferior breeding of alleles. varietiesForwitheachlowSNP,  MC andof supe- ratioDR. the high  riorto inferior Herein, alleles the alleles thatin theassociated  wereelite inbred withlines  higherwasDRdefined and loweras MCthe values  number wereinbred lines of defined  containing superior as the superioralleles  alleles; divided the conversely, by thetotal  alleles number with  associated lowerlines.  of inbred DR andAmong highertheMC 30 elite  inbred valueslines,the superior  were designated allele as the ratio inferior for the  alleles. nineSNP,  For eachsignificant  the ratio of SNPsranged superior from 10% to inferior alleles in the  (SYN11290 andelite inbred lines  SYN30412) towas  90%defined of inbred as the number(Figure  (PZE-102053209) Forcontaining 4).linesseven SNPssuperior  associated allelesdivided bythetotal number of inbredlines. Amongthe30 
 with CDR traits, the superior allele ratios of three loci (PZE-103097076, PZE-102053209,elite inbredlines,the su- 
perior allele ratio for the nine significant SNPs ranged from 10% (SYN11290 and SYN30412) 
 andPZE-101122473) exceeded 50%, and the ratios of superior alleles for the other three 
to 90% (PZE-102053209) (Figure 4). For seven SNPs associated with CDR traits, the superior  lociallele 
 (SYN38588, SYN11290, and SYN30412) were less than 20% (Figure 4). In addition, 
ratios of three loci (PZE-103097076, PZE-102053209, and PZE-101122473) exceeded  three 50%, SNPs(SYN15586,  and the ratios of superior allelesand PZB01400.1,SYN8680) for the associated other threewith KMC loci (SYN38588, all hadand SYN11290, low su-  perior alleleratios,whichwere less than 
SYN30412) were less than 20% (Figure 4). In addition, three SNPs (SYN15586, PZB01400.1, the 20%(Figure 4). These results indicatedthat  superioralleles of and SYN8680)CDR andwith associated KMC at physiological  KMC maturity  all had low superiorwere allele not widely ratios,which wereusedlessin maize  breeding than 20%among (Figurethe 4).30eliteresults  These inbredlines. Therefore, indicatedmore superior that the superioralleles  alleles of CDR and related  KMC atto high physiological maturitywerenotwidely used in maize breeding 
 DR and low MC need to be integrated into these elite inbred lines, which would be ofamong the 30 eliteinbred lines. Therefore,  practical significancemoreforsuperior allelesthe  improvingrelated  dehydrationand low MC need to high DRcharacteristics to be integrated  of maize at physiolog- 
into these elite inbred lines, which would be of practical significance for improving the  ical maturity. dehydration characteristics of maize at physiological maturity.  Figure Superiorallele 
 4. 4.Superior Figure allele distributions distributions ofofnine  nineSNPs  SNPs30 30  in inelite inbred elite inbred Red Red  lines.lines. and white colors colors and white represent  represent superiorandinferior  superiorand alleles,respectively.  inferior alleles,respectively. 
3. Discussion 
3.1. Dissecting the Genetic Basis of Moisture Changes at Physiological Maturity in Maize Using GWAS 
Abundant phenotypic variation is an important factor for the successful analysis 
of the genetic basis of target traits. Population structure analysis showed that the asso- 
ciation panel used in this study had high genetic diversity, including a tropical group, 
a stiff stalk (SS) group and a non-stiff stalk (NSS) group [35]. The MC and DR values 
ranged from 11.46% to 78.75% and 0.05% to 4.78% (with an average of 0.92%), respectively, 
suggesting that this panel was appropriate for GWAS. In addition, the mean value of 
H2 among 12 traits was 57.34%, which indicated that genetic factors were primary for 
these traits. Thus, it is feasible to decompose the genetic control of moisture changes  

##### Plants 2022, 11, 1989 7 of 11 
at physiological maturity stage using a GWAS strategy. Moreover, a total of nine SNPs 
were associated with CDR45–50, CDR50–55, and KMC45, among which five SNPs were 
located within (or were close to) the kernel dehydration-associated QTL in maize reported 
in previous studies (Table S5). For instance, SYN38588 and PZE-101122473 identified in 
this study were situated in the intervals of QTL (PZE-107014666/PZE-107082484 and 
PZE-101120411/PZE-101128157, respectively) that controlled dehydration initial moisture 
and initial time in maize kernels [11]. PZB01400.1 and SYN8680 were both contained within 
kernel dehydration initial moisture-associated QTL (PZE-101235343/SYN25670) [11]. The 
distance between the significant SNP SYN30412 identified in this study and the kernel 
final moisture-associated QTL (SYN30432/PZE-105040767) reported by Yin et al. was 
0.08 Mb [11]. For the significant SNP PZB01400.1 identified in this study, a nearby (312 bp) 
significant SNP_286449826 associated with KMC at 40 DAP was detected by Li et al. [25]. 
These findings suggested that the genetic loci detected in this study were reliable. Notably, 
the lead SNP SYN38588 (p = 9.58 × 10−11 ) was repeatedly detected for traits CDR45–55 
and CDR50–55. Based on this SNP, we identified two further priority genes controlling 
moisture changes. A candidate gene association study revealed that three significant SNPs 
located in the coding sequence of Zm00001d020615 were associated with CDR50–55. In 
addition, we identified the favorable haplotype AATTAA, which should be given priority in 
marker-assisted selection breeding to cultivate maize varieties with low MC and high DR. 
3.2. Candidate Genes Involved in MC and DR at Physiological Maturity Stage 
At physiological maturity, the MC and DR of maize are mainly determined by two 
factors: stress dehydration caused by field environmental conditions and physiological de- 
hydration caused by seed development and maturation [25]. Some specialized proteins that 
inhibit early embryo germination and avoid damages to embryo development play crucial 
roles in the process of grain physiological dehydration. In the present study, combining 
GWAS and candidate gene association analysis, we identified a key gene Zm00001d020615, 
which encodes a PG. Based on previous research, most of the PGs in plants belong to 
cell-wall localized pectin degrading enzymes and are expressed in different tissues and 
development stages [36]. PGs are involved in seed development, fruit softening, organ 
shedding, pollen ripening, and adversity stress [36]. To control water inflow and exit in 
soybean seeds, the PG encoding gene PG031 affects seed coat permeability by regulating the 
intracellular space of parenchyma tissue and maintaining the integrity of the osteosclerosis 
layer [30]. In tomato, the PG gene SlPG has been confirmed as a key gene for improving 
fruit firmness. After mutagenesis of the SlPG gene, the hydrolysis of cell wall polymers 
decreased, which led to water loss reduction and delay in the softening of tomato fruit [37]. 
In rice, the PSL1 gene functions as a PG that plays an important role in modifying cell wall 
biosynthesis of root and leaf and reducing water loss under drought stress [38]. 
3.3. Application of Superior Alleles in Breeding Maize Varieties with Low MC and High DR 
In our study, SNP markers were provided for breeding new maize varieties with low 
MC and high DR at physiological maturity. By analyzing the utilization of superior alleles in 
30 elite maize inbred lines, the superior allele ratios of three SNPs were found to be greater 
than 50%—one, PZE-102053209, was 90% (Figure 4). This indicated that the superior alleles 
of the three SNP were well maintained by artificial selection. A possible reason is that these 
alleles are closely linked with the agronomic traits of interest to breeders. Owing to natural 
variation in the association panel, researchers can develop molecular markers for breeding 
ideal maize varieties. In addition, only six lines contained more than five superior alleles 
(Figure 4), indicating that breeders have probably paid little attention to maize moisture 
changes at the physiological maturity stage. Therefore, the proportion of favorable alleles in 
30 elite lines should be improved in cultivating maize varieties with low MC and high DR 
by marker-assisted selection breeding. The lines Mo17, Chuan273, Zheng22, and SCLM202, 
containing six superior alleles, are considered as excellent resources for breeding maize 
varieties with low MC and high DR through backcross breeding. In future research, the  

##### Plants 2022, 11, 1989 8 of 11 
function of Zm00001d020615 needs, firstly, to be verified. Then, gene Zm00001d020615 can 
be overexpressed/knocked out to create maize lines with high DR. 
4. Materials and Methods 
4.1. Plant Materials 
 The association panel consisted of 241 diverse maize inbred lines (Table S6), which 
were collected from the southwest China breeding program [35]. These lines were planted 
in greenhouses using a randomized complete block design with three replicates. The 
greenhouse parameters were set as follows: light/darkness = 16/8 h; temperature under 
light/temperature in darkness = 25/22 ◦ C; relative humidity = 65%. In each replicate, every 
line was grown in one row with 14 plants. The distance between two rows was 0.7 m, and 
the row length was 3 m. To ensure the consistency of sampling, we selected the plants that grew uniformly for investigating the target traits. 
4.2. Phenotypic Collection and Data Analysis 
 For each line, the fresh weights (FWs) of the cob and kernel were measured sep- 
arately on the 45th, 50th, and 55th DAP. Then, the dry weights (DWs) of the cob and 
kernel were measured after drying at 80 ◦ C in an oven for 72 h. Three individuals 
were weighed for every line at each stage. The MC was calculated as follows: mois- 
ture content (%) = (FW − DW)/FW × 100%. According to the above method, cob moisture 
content (CMC) and kernel moisture content (KMC) were calculated on the 45th, 50th, and 
55th DAP, respectively. Based on the MC values at two successive stages, DR was cal- 
culated as follows: dehydration rate (45–50 d) (%) = (MC on 45 d-MC on 50 d)/5 × 100%; 
dehydration rate (50–55 d) (%) = (MC on 50 d-MC on 55 d)/5 × 100%; dehydration 
rate (45–55 d) (%) = (MC on 45 d-MC on 55 d)/10 × 100%. The mean value across three 
replicates was used as the final phenotype value. The KMCs on 45, 50, and 55 DAP 
were designated as KMC45, KMC50, and KMC55. The CMCs on 45, 50, and 55 DAP 
were designated as CMC45, CMC50, and CMC55. The kernel DRs for the three time 
spans (namely, 45–50, 50–55, and 45–55 DAP) were denoted as KDR45–50, KDR50–55, and 
KDR45–55. The cob DRs for the three time spans (namely, 45–50, 50–55, and 45–55 DAP) 
were denoted as CDR45–50, CDR50–55, and CDR45–55. Descriptive statistical analysis 
was performed using SPSS25 software. The broad-sense heritability (H2 ) was calculated as 
follows [39]: H2 = σG 2 /σP 2 , σG 2 = (MSG-MSE)/rep, σP 2 = (MSG-MSE)/rep + MSE. Herein, 
σG 2 , σP 2 , MSG, MSE, and rep represent genotypic variance, phenotypic variance, mean 
square of genotype, mean square of error, and number of replicates, respectively. 
4.3. Genome-Wide Association Study 
In our previous study, genotyping of the maize panel was performed by Illumina 
Maize SNP50K Bead Chip, and a total of 56,110 SNPs were detected [35]. According to fil- 
tration criteria of missing rate >20% and minor allele frequency (MAF) ≤ 0.05, 46,603 SNPs 
were retained for GWAS. Cluster analysis revealed that the 241 inbred lines could be divided 
into three types: a tropical group (population 1), a non-stiff stalk (NSS) group (population 2), 
and a stiff stalk (SS) group (population 3) based on the 46,603 SNPs (Figure S2). As the 
FarmCPU model was superior to other models (general linear model and mixed linear 
model) in balancing false positives/negatives [40], we used the FarmCPU to detect the as- 
sociations for moisture changes. The model was executed by the R Studio ver.4.0.3 (Allaire, 
Boston, MA, USA) with a FarmCPU package [41]. The simpleM program in R Studio 
ver.4.0.3 (Allaire, Boston, MA, USA) [42] was applied to calculate the effective marker 
number of independent tests (Meff_G = 27,507) [42–44]. The significance threshold was 
calculated as p = 0.05/Meff_G = 1.82 × 10−6 [26]. In addition, the gene models that were lo- 
cated in the linkage disequilibrium (LD) region of each SNP were considered the candidate 
genes. Combined with the gene functional annotations from NCBI (NCBI, National Center 
for Biotechnology Information, https://www.ncbi.nlm.nih.gov/, accessed on 24 February 
2022) databases, we determined the hub genes which were related to moisture changes.  

##### Plants 2022, 11, 19899 of 11 
4.4. Candidate Gene Association Study 
We amplified the hub gene sequences, including the promoter (upstream 2000 bp), 
UTR, and gene body regions in 67 inbred lines, randomly selected from three subgroups 
of the association panel (Figure S3). The PCR-amplified sequences were aligned with 
the B73 (v4) genome using DNAMAN ver.5.2.2 (Reachsoft, Beijing, China) [45]. The 
variations (SNPs and InDels) with MAF ≥ 5%, and the trait phenotypes, were used as 
the inputs for a candidate gene association study based on a general linear model in 
TASSEL ver.5.0 (Buckler lab, Cornell university, New York, NY, USA) software [23]. The 
significant threshold was set as: p = 0.05/n (n represents the number of SNPs). The LD 
decay between pairwise SNPs was calculated using Haploview.JRE ver.18.0.2 (The Broad 
Institute of MIT and Harvard, Cambridge, MA, USA) software [46]. Phenotypic differences between haplotypes were analyzed using a t-test. 
4.5. Analysis of Superior Alleles 
In our study, alleles associated with low MC and high DR were considered as superior 
alleles. The ratio of superior alleles for each significant SNP was calculated as the number 
of lines with superior alleles divided by the total number of lines [19]. According to the 
rate of superior alleles in each line, we generated a visualized heat map using the heatmap 
package in R Studio ver.4.0.3 (Allaire, Boston, MA, USA) software. 
 Supplementary Materials: The following supporting information can be downloaded at: https:// 
www.mdpi.com/article/10.3390/plants11151989/s1, Figure S1: The phenotypic distributions of 
12 collected traits at different time points. The CMC45, CMC50, and CMC55 represent cob moisture 
 content on the 45th, 50th, and 55th days after pollination, respectively. The CDR45–50, CDR50–55, 
 and CDR45–55 represent cob dehydration rate of three time spans: 45–50, 50–55, and 45–55 days after 
pollination, respectively. The KMC45, KMC50, and KMC55 represent kernel moisture content on the 
 45th, 50th, and 55th days after pollination, respectively. The KDR45–50, KDR50–55, and KDR45–55 
 represent kernel dehydration rate of three time spans: 45–50, 50–55, and 45–55 days after pollination, 
 respectively. * p = 0.05, ** p = 0.01, *** p = 0.001; Figure S2: Population structure of 241 maize inbred 
 lines estimated from 46,603 SNPs. Population 1, population 2, and population 3 represent tropical 
 group, non-stiff stalk (NSS) group, and stiff stalk (SS) group, respectively; Figure S3: Phylogenetic 
 tree of 67 maize inbred lines. The group 1, group 2, and group 3 represent tropical group, non-stiff 
 stalk (NSS) group and stiff stalk (SS) group, respectively; Table S1: Significant SNPs detected by 
 GWAS using FarmCPU. CDR45–55 and CDR50–55 represent cob dehydration rate on the 45–55 and 
 50–55 days after pollination, respectively. KMC45 represent kernel moisture content on the 45th, 50th, 
 and 55th days after pollination, respectively; Table S2: Annotations of candidate genes based on B73 
(RefGen_v4) genome. CDR45–55 and CDR50–55 represent cob dehydration rate on the 45–55 and 
 50–55 days after pollination, respectively. KMC45 represent kernel moisture content on the 45th, 50th, 
 and 55th days after pollination, respectively; Table S3: Homologous annotations of 11 unknown genes. 
‘-’ represents no homologous or annotations; Table S4: Details of 38 variations detected in two genes. 
‘-’ represents deletion; Table S5: Significant SNPs overlapping with QTL for kernel moisture change 
 identified in previous studies. CDR45–55 and CDR50–55 represent cob dehydration rate on the 45–55 
 and 50–55 days after pollination, respectively. KMC45 represents kernel moisture content on the 45th 
 day after pollination. mini , dehydration initial moisture. mfni , final moisture. tini , dehydration initial 
 time (DAP). KMC 40 DAP, kernel moisture content on the 40th day after pollination. “–” represents 
 SNP located within the QTL; Table S6: Information of 241 maize inbred lines in the association panel. 
Author Contributions: Conceptualization, L.M. and Y.S.; methodology, M.Z.; software, L.M. and 
M.Z.; validation, M.Z., C.X. and J.Y.; formal analysis, Y.C.; investigation, M.Z., C.X. and H.L.; re- 
sources, Y.S. and G.P.; data curation, C.Z.; writing—original draft preparation, M.Z.; writing—review 
and editing, L.M. and Y.S.; visualization, M.Z.; supervision, L.M. and Y.S.; project administra- 
tion, L.M.; funding acquisition, Y.S. All authors have read and agreed to the published version of the manuscript. 
Funding: This work is supported by the National Key Research and Development Program of China (2021YFF1000303).  

##### Plants 2022, 11, 198910 of 11 Institutional Review Board Statement: Not applicable. Informed Consent Statement: Not applicable. 
Data Availability Statement: The genotype data used in this study were described in a previous 
study. All datasets are available from the corresponding author on reasonable request. 
Acknowledgments: We would like to thank the Maize Research Institute of Sichuan Agricultural 
University for providing the platform. We also extend thanks to other graduate students who attended this project. 
Conflicts of Interest: The authors declare no conflict of interest. References 
1.Wang, G.; Xu, Z.; Wang, F.; Huang, Y.; Xin, Y.; Liang, S.; Li, B.; Si, H.; Sun, L.; Wang, Q.; et al. Development of an efficient and 
precise adenine base editor (ABE) with expanded target range in allotetraploid cotton (Gossypium hirsutum). BMC Biol. 2022, 20, 45. [CrossRef] 
2.Qian, Y.L.; Zhang, X.Q.; Wang, L.F.; Chen, J.; Chen, B.R.; Lv, G.H.; Wu, Z.C.; Guo, J.; Wang, J.; Qi, Y.C.; et al. Detection of QTLs 
controlling fast kernel dehydration in maize (Zea mays L.). Genet. Mol. Res. GMR 2016, 15, gmr.15038151. [CrossRef] 
3.Kang, M.S.; Zhang, S. Narrow-Sense Heritability for and Relationship between seed imbibition and grain moisture loss rate in maize. J. New Seeds 2001, 3, 1–16. [CrossRef] 
4.Shaw, R.H.; Loomis, W.E. Bases for the prediction of corn yields. Plant Physiol. 1950, 25, 225–244. [CrossRef] 
5.Capelle, V.; Remoué, C.; Moreau, L.; Reyss, A.; Mahé, A.; Massonneau, A.; Falque, M.; Charcosset, A.; Thévenot, C.; Rogowsky, P.; 
et al. QTLs and candidate genes for desiccation and abscisic acid content in maize kernels. BMC Plant Biol. 2010, 10, 2. [CrossRef] 
6.Gao, S.; Ming, B.; Li, L.; Yin, X.; Xue, J.; Wang, K.; Xie, R.; Li, S. Relationship and distribution of in-field dry-down and equilibrium 
in maize grain moisture content. Agric. For. Meteorol. 2021, 304–305, 108409. [CrossRef] 
7.Xiang, K.; Reid, L.M.; Zhang, Z.-M.; Zhu, X.-Y.; Pan, G.-T. Characterization of correlation between grain moisture and ear rot 
resistance in maize by QTL meta-analysis. Euphytica 2012, 183, 185–195. [CrossRef] 
8.Zhang, F.; Wang, Y.; Ding, Y.; Zhang, J.; Zhao, X.; Zhao, F.; Tang, B. Corn kernel dehydration rate: Research progress. J. Agric. 2018, 8, 4–8. 
9.Magari, R.; Kang, M.S.; Zhang, Y. Genotype by environment interaction for ear moisture loss rate in corn. Crop Sci. 1998, 37, 774–779. 
10. Schmidt, J.L.; Hallauer, A.R. Estimating harvest date of corn in the field. Crop Sci. 1966, 6, 227–231. [CrossRef] 
11. Yin, S.; Liu, J.; Yang, T.; Li, P.; Xu, Y.; Fang, H.; Xu, S.; Wei, J.; Xue, L.; Hao, D.; et al. Genetic analysis of the seed dehydration 
process in maize based on a logistic model. Crop J. 2020, 8, 182–193. [CrossRef] 
12. Liu, J.; Yu, H.; Liu, Y.; Deng, S.; Liu, Q.; Liu, B.; Xu, M. Genetic dissection of grain water content and dehydration rate related to 
mechanical harvest in maize. BMC Plant Biol. 2020, 20, 118. [CrossRef] [PubMed] 
13. Holland, J.B. Genetic architecture of complex traits in plants. Curr. Opin. Plant Biol. 2007, 10, 156–161. [CrossRef] 
14. Sukumaran, S.; Dreisigacker, S.; Lopes, M.; Chavez, P.; Reynolds, M.P. Genome-wide association study for grain yield and related 
traits in an elite spring wheat population grown in temperate irrigated environments. Theor. Appl. Genet. 2015, 128, 353–363. [CrossRef] [PubMed] 
15. Liu, M.; Tan, X.; Yang, Y.; Liu, P.; Zhang, X.; Zhang, Y.; Wang, L.; Hu, Y.; Ma, L.; Li, Z.; et al. Analysis of the genetic architecture of 
maize kernel size traits by combined linkage and association mapping. Plant Biotechnol. J. 2020, 18, 207–221. [CrossRef] [PubMed] 
16. Zhang, X.; Guan, Z.; Li, Z.; Liu, P.; Ma, L.; Zhang, Y.; Pan, L.; He, S.; Zhang, Y.; Li, P.; et al. A combination of linkage mapping and 
gwas brings new elements on the genetic basis of yield-related traits in maize across multiple environments. Theor. Appl. Genet. 2020, 133, 2881–2895. [CrossRef] 
17. Li, C.; Huang, Y.; Huang, R.; Wu, Y.; Wang, W. The genetic architecture of amylose biosynthesis in maize kernel. Plant Biotechnol. 
J. 2018, 16, 688–695. [CrossRef] 
18. Tian, F.; Bradbury, P.J.; Brown, P.J.; Hung, H.; Sun, Q.; Flint-Garcia, S.; Rocheford, T.R.; McMullen, M.D.; Holland, J.B.; Buckler, 
E.S. Genome-wide association study of leaf architecture in the maize nested association mapping population. Nat. Genet. 2011, 43, 159–162. [CrossRef] 
19. Ma, L.; Zhang, M.; Chen, J.; Qing, C.; He, S.; Zou, C.; Yuan, G.; Yang, C.; Peng, H.; Pan, G.; et al. GWAS and WGCNA uncover 
hub genes controlling salt tolerance in maize (Zea mays L.) seedlings. Theor. Appl. Genet. 2021, 134, 3305–3318. [CrossRef] 
20. Ma, L.; An, R.; Jiang, L.; Zhang, C.; Li, Z.; Zou, C.; Yang, C.; Pan, G.; Lübberstedt, T.; Shen, Y. Effects of ZmHIPP on lead tolerance 
in maize seedlings: Novel ideas for soil bioremediation. J. Hazard. Mater. 2022, 430, 128457. [CrossRef] 
21. Wu, X.; Feng, H.; Wu, D.; Yan, S.; Zhang, P.; Wang, W.; Zhang, J.; Ye, J.; Dai, G.; Fan, Y.; et al. Using high-throughput multiple 
optical phenotyping to decipher the genetic architecture of maize drought tolerance. Genome Biol. 2021, 22, 185. [CrossRef] [PubMed] 
22. Zhang, Y.; Liu, P.; Zhang, X.; Zheng, Q.; Chen, M.; Ge, F.; Li, Z.; Sun, W.; Guan, Z.; Liang, T.; et al. Multi-locus genome-wide 
association study reveals the genetic architecture of stalk lodging resistance-related traits in maize. Front. Plant Sci. 2018, 9, 611. [CrossRef] [PubMed]  

##### Plants 2022, 11, 1989 11 of 11 
23. Pace, J.; Gardner, C.; Romay, C.; Ganapathysubramanian, B.; Lübberstedt, T. Genome-wide association analysis of seedling root 
development in maize (Zea mays L.). BMC Genom. 2015, 16, 47. [CrossRef] [PubMed] 
24. Vanous, A.; Gardner, C.; Blanco, M.; Martin-Schwarze, A.; Lipka, A.E.; Flint-Garcia, S.; Bohn, M.; Edwards, J.; Lübberstedt, T. 
Association mapping of flowering and height traits in germplasm enhancement of maize doubled haploid (GEM-DH) lines. Plant Genome 2018, 11, 170083. [CrossRef] [PubMed] 
25. Li, W.; Yu, Y.; Wang, L.; Luo, Y.; Peng, Y.; Xu, Y.; Liu, X.; Wu, S.; Jian, L.; Xu, J.; et al. The genetic architecture of the dynamic 
changes in grain moisture in maize. Plant Biotechnol. J. 2021, 19, 1195–1205. [CrossRef] [PubMed] 
26. Zhang, Y.; Hu, Y.; Guan, Z.; Liu, P.; He, Y.; Zou, C.; Li, P.; Gao, S.; Peng, H.; Yang, C.; et al. Combined linkage mapping and 
association analysis reveals genetic control of maize kernel moisture content. Physiol. Plant. 2020, 170, 508–518. [CrossRef] 
27. Li, S.; Zhang, C.; Lu, M.; Yang, D.; Qian, Y.; Yue, Y.; Zhang, Z.; Jin, F.; Wang, M.; Liu, X.; et al. QTL mapping and GWAS for field 
kernel water content and kernel dehydration rate before physiological maturity in maize. Sci. Rep. 2020, 10, 13114. [CrossRef] 
28. Li, S.; Zhang, C.; Yang, D.; Lu, M.; Qian, Y.; Jin, F.; Liu, X.; Wang, Y.; Liu, W.; Li, X. Detection of QTNs for kernel moisture 
concentration and kernel dehydration rate before physiological maturity in maize using multi-locus GWAS. Sci. Rep. 2021, 11, 1764. [CrossRef] 
29. Luo, X.; Wang, B.; Gao, S.; Zhang, F.; Terzaghi, W.; Dai, M. Genome-wide association study dissects the genetic bases of salt 
tolerance in maize seedlings. J. Integr. Plant Biol. 2019, 61, 658–674. [CrossRef] 
30. Wang, F.; Sun, X.; Liu, B.; Kong, F.; Pan, X.; Zhang, H. A polygalacturonase gene PG031 regulates seed coat permeability with a 
pleiotropic effect on seed weight in soybean. Theor. Appl. Genet. 2022, 135, 1603–1618. [CrossRef] 
31. Klaumann, S.; Nickolaus, S.D.; Fürst, S.H.; Starck, S.; Schneider, S.; Ekkehard Neuhaus, H.; Trentmann, O. The tonoplast copper 
transporter COPT5 acts as an exporter and is required for interorgan allocation of copper in Arabidopsis Thaliana. New Phytol. 2011, 192, 393–404. [CrossRef] [PubMed] 
32. Xiong, Z.T.; Wang, H. Copper toxicity and bioaccumulation in chinese cabbage (Brassica pekinensis Rupr.). Environ. Toxicol. 2005, 20, 188–194. [CrossRef] [PubMed] 
33. Ahsan, N.; Lee, D.G.; Lee, S.H.; Kang, K.Y.; Lee, J.J.; Kim, P.J.; Yoon, H.S.; Kim, J.S.; Lee, B.H. Excess copper induced physiological 
and proteomic changes in germinating rice seeds. Chemosphere 2007, 67, 1182–1193. [CrossRef] [PubMed] 
34. Mir, A.R.; Pichtel, J.; Hayat, S. Copper: Uptake, toxicity and tolerance in plants and management of Cu-contaminated soil. 
Biometals: Int. J. Role Met. Ions Biol. Biochem. Med. 2021, 34, 737–759. [CrossRef] 
35. Zhang, X.; Zhang, H.; Li, L.; Lan, H.; Ren, Z.; Liu, D.; Wu, L.; Liu, H.; Jaqueth, J.; Li, B.; et al. Characterizing the population 
structure and genetic diversity of maize breeding germplasm in southwest China using genome-wide SNP markers. BMC Genom. 2016, 17, 697. [CrossRef] 
36. Dou, X. Advances in studies on the function of plant polygalacturonase. J. Shanxi Agric. Univ. Nat. Sci. Ed. 2004, 24, 5. 
37. Nie, H.; Shi, Y.; Geng, X.; Xing, G. CRISRP/Cas9-mediated targeted mutagenesis of tomato polygalacturonase gene (SlPG) delays fruit softening. Front. Plant Sci. 2022, 13, 729128. [CrossRef] 
38. Zhang, G.; Hou, X.; Wang, L.; Xu, J.; Chen, J.; Fu, X.; Shen, N.; Nian, J.; Jiang, Z.; Hu, J.; et al. Photo-sensitive leaf rolling 1 encodes 
a polygalacturonase that modifies cell wall structure and drought tolerance in rice. New Phytol. 2021, 229, 890–901. [CrossRef] 
39. Liang, T.; Qing, C.; Liu, P.; Zou, C.; Yuan, G.; Pan, G.; Shen, Y.; Ma, L. Joint GWAS and WGCNA uncover the genetic control of 
calcium accumulation under salt treatment in maize seedlings. Physiol. Plant. 2022, 174, e13606. [CrossRef] 
40. Ma, L.; Liu, M.; Yan, Y.; Qing, C.; Zhang, X.; Zhang, Y.; Long, Y.; Wang, L.; Pan, L.; Zou, C.; et al. Genetic dissection of 
maize embryonic callus regenerative capacity using multi-locus genome-wide association studies. Front. Plant Sci. 2018, 9, 561. [CrossRef] 
41. Kaler, A.S.; Gillman, J.D.; Beissinger, T.; Purcell, L.C. Comparing different statistical models and multiple testing corrections for 
association mapping in soybean and maize. Front. Plant Sci. 2019, 10, 1794. [CrossRef] [PubMed] 
42. Liu, X.; Huang, M.; Fan, B.; Buckler, E.S.; Zhang, Z. Iterative usage of fixed and random effect models for powerful and efficient 
genome-wide association studies. PLoS Genet. 2016, 12, e1005767. [CrossRef] [PubMed] 
43. Gao, X.; Becker, L.C.; Becker, D.M.; Starmer, J.D.; Province, M.A. Avoiding the high Bonferroni penalty in genome-wide association 
studies. Genet. Epidemiol. 2010, 34, 100–105. [CrossRef] [PubMed] 
44. Johnson, R.C.; Nelson, G.W.; Troyer, J.L.; Lautenberger, J.A.; Kessing, B.D.; Winkler, C.A.; O’Brien, S.J. Accounting for multiple 
comparisons in a genome-wide association study (GWAS). BMC Genom. 2010, 11, 724. [CrossRef] 
45. Gao, X.; Starmer, J.; Martin, E.R. A multiple testing correction method for genetic association studies using correlated single 
nucleotide polymorphisms. Genet. Epidemiol. 2008, 32, 361–369. [CrossRef] 
46. Zhang, X.; Guan, Z.; Wang, L.; Fu, J.; Zhang, Y.; Li, Z.; Ma, L.; Liu, P.; Zhang, Y.; Liu, M.; et al. Combined GWAS and QTL analysis 
for dissecting the genetic architecture of kernel test weight in maize. Mol. Genet. Genom. MGG 2020, 295, 409–420. [CrossRef]  